generic_config_updater/field_operation_validators.py

```diff
@@ -20,7 +58,38 @@ def rdma_config_update_validator():
     if branch_version is None:
         return False
 
-    if asic_type == 'cisco-8000':
-        return branch_version >= "20201200"
+    if os.path.exists(GCU_TABLE_MOD_CONF_FILE):
+        with open(GCU_TABLE_MOD_CONF_FILE, "r") as s:
+            gcu_field_operation_conf = json.load(s)
+    else:
+        raise GenericConfigUpdaterError("GCU table modification validators config file not found")
+
+    match = re.search(r'\/([^\/]+)(\/|$)', cleaned_path) # This matches the table name in the path, eg if path if /PFC_WD/GLOBAL, the match would be PFC_WD
```

Comment:
>search [](http://example.com/codeflow?start=15&length=6)

Suggest use library instead of manually parsing.

Reply:
Changed to use jsonpointer library